Sources/SyndiKit/Formats/Media/Podcast/PodcastLocation.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+/// <podcast:location geo="geo:30.2672,97.7431" osm="R113314">Austin, TX</podcast:location>
+public struct PodcastLocation: Codable {
+  public let geo: String
+  public let osm: String
```

Comment:
These have specific specs attached to them. How do you feel about implementing those?
https://github.com/Podcastindex-org/podcast-namespace/blob/main/location/location.md#geo-recommended
https://github.com/Podcastindex-org/podcast-namespace/blob/main/location/location.md#osm-recommended